src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/models/FormLayoutElement.java

```diff
@@ -14,6 +14,8 @@
   private String id;
   private HashMap<String, String> dataModelBindings;
   private TextResourceBindings textResourceBindings;
+  private String optionsId;
+  private Options[] options;
```

Comment:
Another option here is to use a `List<Option>`. I guess this would be more consistent with the other list types.

Reply:
Done